### PR TITLE
feat(nd): terrain on ND

### PR DESCRIPTION
**Pull request by svengcz**
## Summary of Changes

This PR introduces the TERR ON ND functionality.
Other PRs are required to be merged before this one

- [x] Merge map-management in simbridge
- [x] Release simbridge
- [x] Integrate simbridge into A32NX
- [x] Remove INOP sticker
- [x] Refactor simbridge-access in ND modules
- [x] Implement map update behaviour

## Screenshots (if necessary)

grafik
grafik
grafik

## References

Different screenshots, photos and videos in the #pilot-feedback channel

## Additional context

Discord username (if different from GitHub):
Sven [de en]

## Testing instructions

- Start flight (cold & dark) -> align IRS -> start TERR ON ND (check if valid)
- Perform a flight and check if map is updated and terrain is visualized correct (maybe feedback of pilots required)

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/ND/elements/TerrainMap.tsx [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    useEffect(() => {
+        if (!terrOnNdActive) {
+            setMapVisualization(new MapVisualizationData());
+        } else if (mapVisualizationRef.current?.RerenderTimeout === undefined) {
+            const newVisualizationData = new MapVisualizationData(mapVisualizationRef.current);
+            newVisualizationData.RerenderTimeout = RerenderingTimeout;
+            setMapVisualization(newVisualizationData);
+        }
+
+        const meterPerPixel = Math.round(rangeSettings[rangeIndex] * 1852 / height);
```
Comment (2hwk) [inline src/instruments/src/ND/elements/TerrainMap.tsx:255]: Reduce magic numbers

**Review by 2hwk — CHANGES_REQUESTED** [review]

File: src/instruments/src/ND/elements/TerrainMap.tsx (hunk 1 above)
Comment (tracernz) [reply src/instruments/src/ND/elements/TerrainMap.tsx:255]: 1852 (metres in a nautical mile) and 0.3048 (feet in a metre) are two you can't help but know off-by-heart when working in these areas... bit like 3600 for time. 😛

**Review by 2hwk — APPROVED** [review]

File: flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml [hunk 2]
```diff
@@ -132,9 +132,9 @@
 
         <UseTemplate Name="FBW_AIRLINER_SubNodes_Setter_Template">
             <TEMPLATE_TO_CALL>ASOBO_GT_Push_Button_Airliner</TEMPLATE_TO_CALL>
-            <LEFT_SINGLE_CODE>(&gt;H:A320_Neo_MFD_BTN_TERRONND_#ID#)</LEFT_SINGLE_CODE>
+            <LEFT_SINGLE_CODE>(L:A32NX_EFIS_TERR_#ID#_ACTIVE) ! (&gt;L:A32NX_EFIS_TERR_#ID#_ACTIVE)</LEFT_SINGLE_CODE>
```
Comment (tracernz) [inline flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml:135]: Please document all new/modified Lvars in docs/simvars.md

**Comment by Hellfire-2345** [comment]
Quality Assurance Trainee Report 

Discord          : hellfire#7383
Object of testing: #7268
Tier of Testing  : 2
Date             : 14/11/2022

Testing Process: 
after climbing on departure the ND showed all correct terrain (checked on nav aids), frequently updated
and continued to show correct data throughout flight. on arrival and decent I clearly showed the terrain.
but I felt as if it should have been showing the ground a little closer but that's just a very minor thing.
It showed the terrain correctly on the approach. 


Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
Aircraft operated correctly to a high standard 

Media:       
Screenshot_20221114_055019
Screenshot_20221115_051827
Screenshot_20221115_052140
Screenshot_20221115_053625
Screenshot_20221115_053822
Screenshot_20221115_054606

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7268
Tier of Testing  : 2
Date             : 22/11/2022

Testing Process: 
Start flight (cold & dark) -> align IRS -> start TERR ON ND (check if valid) - **OK**
Perform a flight and check if map is updated and terrain is visualized correct - **OK**, range issue is fixed

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image
image

**Pull request merged**